app/cafe.py

```diff
@@ -1,8 +1,7 @@
-from app.errors import (NotVaccinatedError,
-                        OutdatedVaccineError,
-                        NotWearingMaskError
-                        )
 import datetime
+from app.errors import OutdatedVaccineError
+from app.errors import NotVaccinatedError
+from app.errors import NotWearingMaskError

```

Comment:
No, It is bad practice. Do:
```suggestion
from app.errors import (
    OutdatedVaccineError,
    NotVaccinatedError, 
    NotWearingMaskError
)
```